tracer/src/Datadog.Trace/Iast/IastModule.cs

```diff
@@ -277,15 +277,17 @@ public static bool AddRequestVulnerabilitiesAllowed()
         }
 
         // Sometimes we do not have the file/line but we have the method/class.
-        var filename = frameInfo.StackFrame?.GetFileName();
+        var stackFrame = frameInfo.StackFrame;
+        var filename = stackFrame?.GetFileName();
+        var line = string.IsNullOrEmpty(filename) ? 0 : (stackFrame?.GetFileLineNumber() ?? 0);
```

Comment:
not sure if it's really worth changing, but I have feeling defaulting to null rather than zero could simplify this code a little bit.